File: debugging-questions/Round2/py/question2.py
# This code reverses the elements of an array.
def reverse_array(arr):
    for i in range(len(arr) // 2 + 1):  
        arr[i], arr[-i] = arr[-i], arr[i]  
    return arr

arr = [1, 2, 3, 4, 5]
print("Reversed array:", reverse_array(arr))

File: debugging-questions/Round2/c/question2.c
// This code reverses the elements of an array.
#include <stdio.h>

void reverse_array(int arr[], int n) {
    for (int i = 0; i <= n / 2; i++) { 
        int temp = arr[i];
        arr[i] = arr[n - i]; 
        arr[n - i] = temp;
    }
}
int main() {
    int arr[] = {1, 2, 3, 4, 5};
    int n = sizeof(arr) / sizeof(arr[0]);
    reverse_array(arr, n);
    printf("Reversed array: ");
    for (int i = 0; i < n; i++) {
        printf("%d ", arr[i]);
    }
    return 0;
}
